src/components/generic/Modal/Modal.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {Modal, ModalBody, ModalHeader} from '@momentum-ui/react';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../../constants';
+
+/**
+ * Modal component
+ *
+ * @param {object} props  Data passed to the component
+ * @param {string} props.className   Custom CSS class to apply
+ * @param {string[]} props.children  List of children
+ * @param {string} props.headerLabel  Title of the modal
+ * @param {Function} props.onHide  Callback when the modal is closed
+ * @param {boolean} props.show  Show or hide the modal
+ * @param {boolean} props.showCloseButton  Show or hide the modal close button
+ *
+ * @returns {object}  JSX of the element
+ */
+export default function ModalComponent({
+  className,
+  children,
+  headerLabel,
+  onHide,
+  show,
+  showCloseButton,
+  ...props
+}) {
+  const mainClasses = {
+    [`${WEBEX_COMPONENTS_CLASS_PREFIX}-modal`]: true,
+    [className]: !!className,
+  };
+  const customId = `${WEBEX_COMPONENTS_CLASS_PREFIX}-${Math.floor((Math.random() * 100000000) + 1)}`;
```

Comment:
call this `testId`, please, and use it for `htmlId` https://github.com/momentum-design/momentum-ui/blob/master/react/src/lib/Modal/index.js#L125